lib/grammers-client/src/lib.rs

```diff
@@ -46,3 +46,9 @@ pub(crate) mod utils;
 
 pub use client::{Client, Config, InitParams, SignInError};
 pub use types::{button, reply_markup, ChatMap, InputMessage, Update};
+
+pub use grammers_crypto;
+pub use grammers_mtproto;
+pub use grammers_mtsender;
+pub use grammers_session;
```

Comment:
I think it's fine to re-export all of this, but making the user type `grammers_client::grammers_session` is a bit of a mouthful. There is only one session. So we could rename this to `pub use grammers_session as session`.